optuna/samplers/_nsga2/sampler.py

```diff
@@ -154,18 +162,36 @@ def sample_relative(
         generation = parent_generation + 1
         study._storage.set_trial_system_attr(trial_id, _GENERATION_KEY, generation)
 
+        dominates = _dominates if self._constraints_func is None else _constrained_dominates
         if parent_generation >= 0:
-            p0 = self._select_parent(study, parent_population)
+            # crossover
```

Comment:
Please write down a sentence like `We choose a child based on the specified crossover method`.